module5/labs/Week4/CallingThirdPartyAPIs.md

```diff
@@ -0,0 +1,50 @@
+---
+layout: page
+title: Calling Third Party APIs Lab
+---
+
+### Overview

```

Comment:
Love this lab!

Reply:
Thanks! Glad we brainstormed together, because I'm very excited about this one too!!